spec/features/shelter_reviews/edit_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe "shelter review edit page" do
+  it "can edit shelter reviews" do
+    shelter_1 = Shelter.create(name: "Happy Puppies", address: "55 Street St", city: "Danger Mountain", state: "UT", zip: "80304")
+
+    review_1 = ShelterReview.create!(title: "What a great place!", rating: "5", content: "We got our new puppy from Happy Puppies and they totally lived up to their name.", image: "image.jpeg", shelter_id: shelter_1.id)
+
+    visit "shelters/#{shelter_1.id}/reviews/#{review_1.id}/edit"
+
+    expect(page).to have_content("Title:")
+    expect(page).to have_content("Rating:")
+    expect(page).to have_content("Content:")
```

Comment:
The instruction is to verify that the fields are auto-populated with the current value. I verified that we are auto-populating on the local server, but I can't solve the riddle of how to test it. I think it is hidden or secret or some such setting 🤷‍♀️

Reply:
Hmm...I'm not sure how one would do that. If you haven't already, perhaps add it to the debugging section in project as a reminder to solve the issue.

Reply:
Perfect, excellent suggestion. I'll create a card and an issue for it :)